historical.js

```diff
@@ -291,7 +293,25 @@ clearFigOptBtn.on("click", function () {
     $("input.figOpt").val('');
 });
 
+//add event listener on nav bar to sync tabs and pills
+$(document).ready(function () {
+    function syncTabs(targetId) {
+      $('.nav-link').removeClass('active');
+      $('[href="' + targetId + '"]').addClass('active');
+      $('.tab-pane').removeClass('show active');
+      $(targetId).addClass('show active');
+    }
 
+    $('#pills-hist .nav-link, #tabs-hist .nav-link').on('click', function (e) {
+      e.preventDefault();
+      var targetId = $(this).attr('href');
+      syncTabs(targetId);
+    });
+
+    // Automatically sync the active tab on page load
+    var activeTabId = $('.nav-link.active').attr('href');
+    syncTabs(activeTabId);
+});
```

Comment:
This is probably not the reason of navpil not working but if this is in one of the .js loaded in the index.html we probably only need one of this in the .js file right?